hackathon_site/event/test_api.py

```diff
@@ -87,3 +88,106 @@ def test_user_has_profile(self):
         serializer = TeamSerializer(team_expected)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.json(), serializer.data)
+
+
+class LeaveTeamTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.team = Team.objects.create()
+
+        self.profile = Profile.objects.create(user=self.user, team=self.team)
+
+        self.view = reverse("api:event:leave-team")
+
+    def test_user_not_logged_in(self):
+        response = self.client.post(self.view)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_user_has_no_profile(self):
+        self.profile.delete()
+        self._login()
+        response = self.client.post(self.view)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_leave_and_delete(self):
+        """
+            when there is no orders or other members on the team,
```

Comment:
Minor, don't indent the docstrings to be consistent with the ones elsewhere. Same for the ones below